docs/docs/command-line-interface.mdx

```diff
@@ -313,31 +313,32 @@ rasa data convert nlg --help
 
 ## rasa data validate
 
-You can check your domain, NLU data, or conversation data for mistakes and inconsistencies. 
+You can check your domain, NLU data, or story data for mistakes and inconsistencies. 
 To validate your data, run this command:
 
 ```bash
 rasa data validate
 ```
 
-By default, the validator searches only for errors in the data, e.g. the same training
-example being listed as an example for two intents.
-To catch minor issues that don't prevent training a model but might indicate messy data
-(e.g. unused intents), use the `--fail-on-warnings` flag.
+By default, the validator searches for errors in the data, e.g. two intents that have some
```

Comment:
```suggestion
The validator searches for errors in the data, e.g. two intents that have some
```
I'd remove the "by default" since we're not bringing up the fail on warnings flag til way later